Providers/Profile.swift

```diff
@@ -763,7 +814,7 @@ open class BrowserProfile: Profile {
 
         // TODO: Do we still need this/do we need to do this for our new DB too?
         private func handleRecreationOfDatabaseNamed(name: String?) -> Success {
-            let browserCollections = ["history", "tabs"]
+            let browserCollections = ["history"]
```

Comment:
should we keep `tabs` here or this is deliberately removed from locally reset collections

Reply:
I removed tabs from this line because I thought `handleRecreationOfDatabaseNamed` was recreating the browserDB databases. But upon closer reading I can see it's just resetting the tabs and history collections so I'll add tabs back.